### PR TITLE
add an OPAM file

### PR DESCRIPTION
**Pull request by palmskog**
Filled in some reasonable info for an OPAM file.

### CODE REVIEW

File: opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "keyword:lemma overloading"
+  "category:Computer Science/Data Types and Data Structures"
+  "logpath:LemmaOverloading"
+]
+authors: [
+  "Georges Gonthier"
+  "Beta Ziliani"
+  "Aleksandar Nanevski"
+  "Derek Dreyer"
+  "Anton Trunov"
```
Comment (anton-trunov) [inline opam]: Thanks for the contribution, but I shouldn't be among the authors. I just slightly adopted the project to Coq 8.8 :)

**Comment by anton-trunov** [comment]
Btw, this project is on OPAM (`extra-dev` category, [here](https://github.com/coq/opam-coq-archive/blob/master/extra-dev/packages/coq-lemma-overloading/coq-lemma-overloading.dev/opam)). 
Do you think it would make sense to move it to `released`?

**Comment by palmskog** [comment]
I think it makes sense to have all coq-community projects that have releases under `released` in the OPAM repo. If the author chooses not to have releases, that's fine by me though. I can submit a package for the 8.8.0 release.

**Comment by anton-trunov** [comment]
> I can submit a package for the 8.8.0 release.

Great! Thank you.

**Pull request merged**